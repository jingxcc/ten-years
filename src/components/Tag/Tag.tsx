interface TagProps {
  content: string;
}

export default function Tag({ content }: TagProps) {
  return (
    <span className="rounded-full border border-sky-300 px-3 py-1">
      {content}
    </span>
  );
}
