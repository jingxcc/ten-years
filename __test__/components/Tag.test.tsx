import renderer from "react-test-renderer";
import Tag from "@/components/Tag/Tag";

it("renders correctly", () => {
  const tree = renderer.create(<Tag content="sample tag"></Tag>).toJSON();
  expect(tree).toMatchSnapshot();
});
