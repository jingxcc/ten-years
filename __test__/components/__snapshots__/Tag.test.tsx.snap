// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span
  className="rounded-full border border-sky-300 px-3 py-1"
>
  sample tag
</span>
`;
